tests/test_async.py

```diff
@@ -4,83 +4,90 @@
     asyncio = None
 
 import gc
-import sys
 import unittest
+from sys import version_info as py_version_info
 
 import pexpect
 from pexpect import replwrap
+
 from .PexpectTestCase import PexpectTestCase
 
+
 def run(coro):
-    return asyncio.get_event_loop().run_until_complete(coro)
+    if py_version_info >= (3, 7):
+        # get_running_loop, new in 3.7, is preferred to get_event_loop
+        return asyncio.get_running_loop().run_until_complete(coro)
+    else:
+        return asyncio.get_event_loop().run_until_complete(coro)
+
 
 @unittest.skipIf(asyncio is None, "Requires asyncio")
 class AsyncTests(PexpectTestCase):
     def test_simple_expect(self):
-        p = pexpect.spawn('cat')
```

Comment:
Please do not include style changes like these, could you please revert these?

Reply:
Ack